.github/workflows/markdown-lint.yml

```
@@ -19,17 +19,23 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v28.0.0
+        with:
+          files: |
+            **/*.md
```

I'd write this the same as in line 10:

```suggestion
          files: '**/*.md'
```